src/arrayMethodSort.js

```diff
@@ -2,10 +2,42 @@
 
 /**
  * Implement method Sort
+ *
+ *
  */
+
+const func = function (a, b) {
+  const aNum = String(a);
+  const bNum = String(b);
+
+  if (aNum < bNum) {
+    return -1;
+  }
+
+  if (aNum > bNum) {
+    return 1;
+  }
+
+  if (aNum === bNum) {
+    return 0;
+  }
+};
```

Comment:
This function does the job, but the name `func` is not informative. 

Consider renaming it to `defaultCompare` or something like that.